Makefile

```diff
@@ -41,12 +41,7 @@ test:
 # TODO: add '-race'
 
 build:
-	GOOS=linux   GOARCH=amd64 CGO_ENABLED=0 go build -ldflags '${LDFLAGS}' -o release/linux/amd64/plugin-git
```

Comment:
we should use `?=` to assign default values to TARGETOS & TARGETARCH

Reply:
For local build command execution?

Reply:
Added default to linux/amd64